src/function/statistics/corr.js

```diff
@@ -13,8 +15,10 @@ export const createCorr = /* #__PURE__ */ factory(name, dependencies, ({ typed,
    * Examples:
    *
    *     math.corr([1, 2, 3, 4, 5], [4, 5, 6, 7, 8])     // returns 1
-   *     math.corr([1, 2.2, 3, 4.8, 5], [4, 5.3, 6.6, 7, 8])     // returns 0.9569941688503644
-   *     math.corr(math.matrix([[1, 2.2, 3, 4.8, 5], [1, 2, 3, 4, 5]]), math.matrix([[4, 5.3, 6.6, 7, 8], [1, 2, 3, 4, 5]])) // returns DenseMatrix [0.9569941688503644, 1]
+   *     math.corr([1, 2.2, 3, 4.8, 5], [4, 5.3, 6.6, 7, 8])     //returns 0.9569941688503644
+   *     math.corr([[1, 2.2, 3, 4.8, 5], [4, 5.3, 6.6, 7, 8]],[[1, 2.2, 3, 4.8, 5], [4, 5.3, 6.6, 7, 8]])   // returns [1,1]
+   *     math.corr(math.matrix([2, 4, 6, 8]), math.matrix([1, 2, 3, 6]))    // returns 0.9561828874675149
```

Comment:
Can you remove the examples with `math.matrix`? They are duplicates of the Array examples, so don't add much

Reply:
I have removed examples with math.matrix